.github/workflows/security.yaml

```
@@ -0,0 +1,32 @@
+---
+ name: security
+
+ on:
+   push:
+     branches: [development/*, stabilization/*, hotfix/*]
+   release:
+     types: [published]
+   pull_request:
+     branches: [development/*, stabilization/*, hotfix/*]
+   schedule:
+     - cron: '0 8 * * 1' # Monday - 8am - UTC
+   workflow_dispatch:
+
+ jobs:
+   build:
+     permissions:
+       contents: read
+       packages: write
+     uses: ./.github/workflows/build.yml
+     secrets:
+       REGISTRY_LOGIN: ${{ secrets.REGISTRY_LOGIN }}
+       REGISTRY_PASSWORD: ${{ secrets.REGISTRY_PASSWORD }}
```

The secrets are badly defined in the workflow, use this method instead:
```suggestion
     secrets: inherit
```